types/stripe/index.d.ts

```diff
@@ -842,10 +870,15 @@ declare namespace Stripe {
              */
             refunds: IChargeRefunds;
 
+            /**
+             * ID of the review associated with this charge if one exists. [Expandable]
+             */
+            review?: string | null;
```

Comment:
This property is `Expandable` which means it can be expanded into an object (see `source_transfer` on L894 as an example). Can you also add the object interface to the type (creating the interface if need be)?